run_dir/design/bioinfo_tab/run_lane_sample_view.html

```diff
@@ -24,41 +24,41 @@
       <th></th>
     </tr>
     <tr class="bioinfo-field-names bioinfo-project" id="bioinfo-project-{{ project_id }}">
-      <th class="bioinfo-project"><a href="#bioinfo-project-{{ project_id }}" class="bioinfo-expand">
+      <th class="bioinfo-project sticky"><a href="#bioinfo-project-{{ project_id }}" class="bioinfo-expand">
```

Comment:
I like this class name! :)

Reply:
But wouldn't it be smoother to put it on the tr-tag instead of all the individual th-tags?